### ISSUE TITLE
Add a platform support page to the docs

### ISSUE THREAD

**Issue by FFY00** [python/cpython#110227]
# Documentation

AFAICT platform support is currently being documented in [PEP 11](https://peps.python.org/pep-0011/).

Perhaps, it would be nice to have a dedicated platform support page in our documentation (eg. [rust](https://doc.rust-lang.org/nightly/rustc/platform-support.html)). I think this would make the information more discoverable and future-proof, as the PEP may be superseded.

PS: I, myself, had to spend a little bit of time on Google to find the PEP.

**Comment by anujrmohite** [comment]
Indeed. it would have been quick to find information but now we have to rely on external dedicated platform support page.

**Comment by AA-Turner** [comment]
I think there's merit providing a link to PEP 11, but as the docs are versioned there are disadvantages to maintaining the information here.

A

**Comment by anujrmohite** [comment]
oh it will potentially be time-consuming to documentation versioning each time ig r8?

**Comment by terryjreedy** [comment]
PEP 11 is not superceded as it is one of the 'low number' (< 20?) PEPs edited as needed.  The last Post-History entry is [10-Mar-2022](https://mail.python.org/archives/list/python-committers@python.org/thread/K757345KX6W5ZLTWYBUXOXQTJJTL7GW5/).  Post dates link to the relevant discussion. 

I agree that this should be linked in the devguide (already done) and also somewhere in the Reference and indexed ('platform support'?).

**Comment by FFY00** [comment]
> I think there's merit providing a link to PEP 11, but as the docs are versioned there are disadvantages to maintaining the information here.

The maintenance commitments may change between versions, so I think it especially makes sense to maintain that information here :sweat_smile:

> PEP 11 is not superceded as it is one of the 'low number' (< 20?) PEPs edited as needed. The last Post-History entry is [10-Mar-2022](https://mail.python.org/archives/list/python-committers@python.org/thread/K757345KX6W5ZLTWYBUXOXQTJJTL7GW5/). Post dates link to the relevant discussion.

Thanks for pointing it out, it was not super clear to me from PEP 1 if this kind of thing should be standard practice. As I understand it now, from your reply, editing key meta-PEPs is standard practice, while substantially editing other PEPs is not, right?

> I agree that this should be linked in the devguide (already done) and also somewhere in the Reference and indexed ('platform support'?).

I think we should at least also link it here, no? The devguide is for contributors, not general users.

**Comment by StanFromIreland** [comment]
A duplicate issue of this was recently closed, I will try and find it.

Note that PEP 11 is already linked on the [configure page](https://docs.python.org/3.15/using/configure.html#build-requirements) (below long list of version changeds).
